### ISSUE TITLE
Navigate to /ask/message/ navigates to channel selection screen

### ISSUE THREAD

**Issue by mnmaraes** [helpscout/beacon-ios-sdk#129]
Trying to navigate to the `email` screen via 
``` Obj-C
[HSBeacon navigate:@"/ask/message/" beaconSettings: settings];
```

https://developer.helpscout.com/beacon-2/ios/#navigate-to-a-specific-screen

* Beacon install method: [ ] Carthage, [x] CocoaPods, [ ] Manual

**Actual outcome**
Navigates to channel selection screen

**Expected outcome**
Navigates to `email` screen

**Screenshots or GIFs**
Any examples of the issue

![Simulator Screen Shot - iPhone 11 - 2020-11-06 at 17 30 43](https://user-images.githubusercontent.com/3777699/98573080-50316380-22ae-11eb-9eb0-a1c479bfbb06.png)

**Comment by ScottPetit** [comment]
Hey @mnmaraes,

Just wanted to let you know that this should be resolved with our latest version (2.1.1). If you want to see what else in this release feel free to check out our [Changelog](https://github.com/helpscout/beacon-ios-sdk/blob/master/CHANGELOG.md).

**Comment by mnmaraes** [comment]
@ScottPetit: Thank you, Scott
